README.md

```diff
@@ -1,3 +1,251 @@
+# The Chowbus Admin
+
+This app managees the assignments of scheduled deliveries for restaurants to designated areas.
+
+## Setup
+
+### Dependencies
+
+1. Ruby 2.4.1
```

Comment:
Do you use any Ruby version managers like rbenv? Usually ruby version is declared in .ruby or Gemfile.

Reply:
Ah, I've been using ```rvm```.

Reply:
I think rvm also works with the .ruby file - just a nice thing to add so that your reviewer (and eventually your team) can setup the correct env more easily.